src/test/java/de/tum/in/www1/artemis/ResultServiceIntegrationTest.java

```diff
@@ -490,6 +505,44 @@ public void getResult_asStudent() throws Exception {
         request.get("/api/results/" + result.getId(), HttpStatus.FORBIDDEN, Result.class);
     }
 
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void testGetResultsForExamExercise() throws Exception {
+        var now = ZonedDateTime.now();
+        for (int i = 1; i <= 10; i++) {
+            ModelingSubmission modelingSubmission = new ModelingSubmission();
+            modelingSubmission.model("Text");
+            modelingSubmission.submitted(true);
+            modelingSubmission.submissionDate(now.minusHours(3));
+            database.addSubmission(this.examModelingExercise, modelingSubmission, "student" + i);
+            if (i % 3 == 0) {
```

Comment:
what's the idea behind only adding results to submission 3, 4, 6, 8, 9 ?

Reply:
not a specific one, just used to have different submissions